src/components/map/KmlService.js

```diff
@@ -153,7 +153,7 @@ goog.require('ga_urlutils_service');
           // OL applies a default scale multiplier to 0.5
           // https://github.com/openlayers/ol3/blob/master/src/ol/format/kml.js#L622
           // https://github.com/openlayers/ol3/pull/5745
-          if (image && image.getScale()) {
+          if (image && image.getScale() && image.getScale() != 1) {
```

Comment:
Are you sure...
Here they used to define the setter:

https://github.com/openlayers/ol3/blob/v3.18.0/src/ol/format/kml.js#L1451 (where they took the square root if scale was defined)

Here the read the node (so they use take the square root):
https://github.com/openlayers/ol3/blob/v3.18.0/src/ol/format/kml.js#L611

then

https://github.com/openlayers/ol3/blob/v3.18.0/src/ol/format/kml.js#L214

I don't think they ever multiplied the scale by 2.
Can you find me the reference?

I would really just do:
```
if (image && image.getScale()) {
    image.setScale(Math.sqrt(image.getScale());
}
```

That is if you want to keep the old behaviour. Maybe I don't understand....